packages/shared/src/server/redis/legacyIngestion.ts

```diff
@@ -21,7 +21,7 @@ export class LegacyIngestionQueue {
             connection: newRedis,
             defaultJobOptions: {
               removeOnComplete: true,
-              removeOnFail: 100_000,
+              removeOnFail: 500_000,
```

Comment:
Currently, we overwrite that to 1 million. I think picking the middle here is a reasonable call.